packages/swirl-tokens/dart/lib/styles.dark.dart

```diff
@@ -4,7 +4,7 @@
 //
 
 // Do not edit directly
-// Generated on Fri, 15 Dec 2023 15:44:18 GMT
```

Comment:
**question:** this is a bit annoying ... can the timestamp be removed somehow?

Reply:
my current state is that this is not possible unfortunately. but the answer for why maybe has @Sqrrl